rfc/rfc014-authorization-credential.md

```diff
@@ -0,0 +1,218 @@
+A policy MAY also require certain parameters. For example, when a `search` operation is done on a FHIR `observation` resource, the policy may have a rule that requires a query parameter using the `subject` field of the credential.
+All restrictions and policy rules MUST use paths relative to the endpoint from the Nuts registry for the given service.
+If `restrictions` are present in the credential, the resource server can compare the operation and relative path of the request to the `restrictions` present in the credential.
+
+Although Nuts Authorization Credentials are part of the OAuth flow of [RFC003](rfc003-oauth2-authorization.md), the actual checking is done at request time. This means that the resource server will have to check the policy and make a request for the restrictions from the Nuts registry. This model can be compared with an Attribute Based Access Control (ABAC) model. The Bolt policies are added to the Policy Administration Point, the Nuts node acts as Policy Information Point. The resource server is the Policy Enforcement Point. It's up to the vendor to implement the Policy Decision Point. 
+
+## 5. Issuance & distribution
+
+A Nuts Authorization Credential is private and the transaction MAY be published over the Nuts network.
+The contents of the Credential MUST NOT be attached to the network transaction.
+Only the custodian and actor MAY request and retrieve the transaction payload.
+Every DID MAY issue an authorization credential.
+The VC does not have any other requirements nor does it add requirements to other VCs.
+
+## 6. Supported proofs
+
+Only proofs from [RFC011](rfc011-verifiable-credential.md) are supported.
+
+## 7. Trust
+
+The Nuts Authorization Credential only impacts the actor and custodian.
+It MUST be trusted automatically.
+
+## 8. Revocation
+
+The Nuts Authorization Credential uses the revocation mechanism as stated by [RFC011](rfc011-verifiable-credential.md).
+
+## 9. Use cases
+
+The Nuts Authorization Credential MUST be used in the Nuts OAuth flow as stated in [RFC003](rfc003-oauth2-authorization.md).
+
+## 10. Privacy considerations
+
+The credential will most likely contain a citizen service number and MUST therefore be private to the actor and custodian.
+
+## 11. Services
+
+A Nuts Authorization Credential is always scoped to a specific Bolt.
+
+## 12. Examples
+
+Example of a Nuts Authorization Credential with explicit consent:
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://nuts.nl/credentials/v1"
+  ],
+  "id": "did:nuts:custodian#90382475609238467",
+  "type": ["VerifiableCredential", "NutsAuthorizationCredential"],
+  "issuer": "did:nuts:EgFjg8zqN6eN3oiKtSvmUucao4VF18m2Q9fftAeANTBd",
+  "issuanceDate": "2010-01-01T19:73:24Z",
+  "expirationDate": "2010-02-01T19:73:24Z",
+  "credentialSubject": {
+    "id": "did:nuts:SjkuVHVqZndMVVJwcnUzbjhuZklhODB1M1M0LW9LcWY0WUs5S2",
+    "legalBase": {
+      "consentType": "explicit",
```

Comment:
I'm wondering if `consent` shouldn't be an element on its own

```json
{
   "consent": "implicit"
}
```

and/or:

```json
{
   "consent": {
       "type": "explicit",
       "evidence": {
           ...
       }
   }
}
```

Can there be other legal bases than implicit/explicit consent?

Is "implicit consent" actually a think?
Edit: it is: https://en.wikipedia.org/wiki/Implied_consent

Reply:
legal base could also be an emergency which doesn't involve patient consent